Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Multiple-Textbox/index.md

```diff
@@ -50,4 +50,50 @@ The Repeatable textstrings property editor enables a content editor to make a li
         </ul>
     }
 }
-```
\ No newline at end of file
+```
+
+## Add values programmatically
+
+See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../../Reference/Management/Services/ContentService/index.md).
+
+```csharp
+@{
+    // Get access to ContentService
+    var contentService = Services.ContentService;
+
+    // Create a variable for the GUID of the page you want to update
+    var guid = new Guid("32e60db4-1283-4caa-9645-f2153f9888ef");
+
+    // Get the page using the GUID you've defined
+    var content = contentService.GetById(guid); // ID of your page
+
+    // Set the value of the property with alias 'keyFeatureList'
+    content.SetValue("keyFeatureList", "Awesome" + Environment.NewLine + "Super");
+
+    // Save the change
+    contentService.Save(content);
+}
+```
+
+:::note
+To add multiple values to the repeatable textstrings property editor you have to put each value on a new line. You can of course also use \r\n\ instead of Environment.NewLine.
```

Comment:
```suggestion
To add multiple values to the repeatable text strings property editor you have to put each value on a new line. This can be achieved using either `\r\n\` or `Environment.NewLine`.
```